src/classes/groundplaces.ts

```diff
@@ -15,6 +20,14 @@ import {
  * @description GroundPlaces business logic.
  */
 export class GroundPlaces {
+  private readonly Generator: Generator = new Generator();
+
+  public readonly Storage: Storage | null = null;
```

Comment:
why cant it be just `public readonly Storage: Storage` ?

Reply:
Done